apps/client/src/app/pages/allagan-reports/model/allagan-report-source.ts

```diff
@@ -1,7 +1,7 @@
 export enum AllaganReportSource {
   FISHING = 'FISHING',
   SPEARFISHING = 'SPEARFISHING',
-  DESYNTH = 'DESYNTH',
+  DESYNTH = 'DESYNTHED_FROM',
```

Comment:
The key is what matters for the enum matches in the rest of the code, it should match the value accordingly.
```suggestion
  DESYNTHED_FROM = 'DESYNTHED_FROM',
```